pkg/cri/sbserver/runtime_config.go

```diff
@@ -0,0 +1,13 @@
+package sbserver
+
+import (
+	"context"
+	"errors"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+// RuntimeConfig returns configuration information of the runtime.
+func (c *criService) RuntimeConfig(ctx context.Context, r *runtime.RuntimeConfigRequest) (*runtime.RuntimeConfigResponse, error) {
+	return nil, errors.New("not implemented")
```

Comment:
Should this use a real "not implemented" error so that the grpc error code uses not implemented. This could get returned back to the client as an unknown error type.

Reply:
Definitely, it is one of the things I want to fix.

If you think this PR overall is something we want, I can fix this and the other small things.